### ISSUE TITLE
Midterm Peer Review

### ISSUE THREAD

**Issue by jmarkus725** [jwxiong/ORIE5741-Project#4]
This project is about predicting stock prices of a subset of S&P 500 companies. The data they are using is public release fundamental information about these companies and inflation data, with their outcome variable being stock prices. Their objective is to build models using this information in order to predict future stock returns, and see how useful this information truly is.

One thing I like about this project is that they have worked to transform their data into potentially more usable forms. The group seems to have a good grasp on the topic and what generally influences stock profitability, so they can understand how best to transform the data to provide the most information.

A second thing I like about this project is that they used multiple types of regression. By comparing these different models, they are able to determine which is best for their data instead of blindly choosing one model and running with it.

A third thing I like about this project is that they used latent information in the data (namely the date of filing) to adjust their model and when they would determine a prediction. It seems to me that this would help stabilize the model since it is looking at constant time intervals instead of random profitability measures.

One thing I think this project can improve upon is potentially reevaluating how they select their training and test sets. As the group described in the report, it does not appear that these sets were randomly chosen, but instead were selected based on year. This might lead to dependencies among the data and affect the model’s accuracy.

A second thing that I think this project can improve upon is considering (or at least documenting) how they are going to try to prevent over- and underfitting of their models. There don’t appear to be a ton of factors present in their models, so I don’t think overfitting is an issue yet, but it is also important to be cognizant of the fact that these features might not adequately predict stock profitability and the models will not be as powerful as they could be with additional variables.

A third thing that I think this project can improve upon is potentially expanding the dataset. It is a little hard to tell, but it looks as if the training set has only a few hundred rows, and I think this might lead to overfitting as the project progresses and the models become more complex. If this is an inaccurate representation of the dataset, then this point is irrelevant.
